### PR TITLE
docs: Expand UI deprecation description

### PR DESCRIPTION
**Pull request by qbatten**
Small docs PR to resolve issue #7038, per convo in office hours on 11/30. Thanks to @Sburwash for pointing out that the Airflow webserver can replace nearly all Meltano UI functionality.

Maintainers please do add or edit, I wasn't sure what other features are/aren't supported by UI, nor do I know the details of how the UI/environments interact, just think I remember someone saying they dont play nice together. Thanks!

### CODE REVIEW

File: docs/src/_guide/troubleshooting.md [hunk 1]
```diff
@@ -122,3 +122,18 @@ Early versions of Meltano promoted a simple UI feature that was used for setting
 Meltano will eventually have a UI again as the company and community grows. Please let us know your thoughts on what you would like to see in a future Meltano UI in [this GitHub discussion](https://github.com/meltano/meltano/discussions/6957).
 
 To view the previous documentation on the UI, review [this pull request](https://github.com/meltano/meltano/pull/6955) where they were removed.
+
+### Limitations & Capabilities of the Deprecated UI
+
+If you are still using the UI, please note that it is not compatible with many newer Meltano features.
+
+The UI **does** work with:
+- Schedules based on ELT (`meltano schedule add <schedule_name> --extractor <tap> --loader <target> --transform ...`)
```
Comment (edgarrmondragon) [inline docs/src/_guide/troubleshooting.md:131]:
```suggestion
- Schedules based on [the `elt` command](https://docs.meltano.com/reference/command-line-interface#elt) (`meltano schedule add <schedule_name> --extractor <tap> --loader <target> --transform ...`)
```

File: docs/src/_guide/troubleshooting.md [hunk 2]
```diff
@@ -122,3 +122,18 @@ Early versions of Meltano promoted a simple UI feature that was used for setting
...
+
+The UI does **not** work with:
+- Schedules based on jobs (`meltano schedule add <schedule_name> --job <job>`)
```
Comment (edgarrmondragon) [inline docs/src/_guide/troubleshooting.md:134]:
```suggestion
- Schedules based on [jobs](https://docs.meltano.com/reference/command-line-interface#job) (`meltano schedule add <schedule_name> --job <job>`)
```

File: docs/src/_guide/troubleshooting.md [hunk 3]
```diff
@@ -122,3 +122,18 @@ Early versions of Meltano promoted a simple UI feature that was used for setting
...
+- Environments
```
Comment (edgarrmondragon) [inline docs/src/_guide/troubleshooting.md:135]:
```suggestion
- [Environments](https://docs.meltano.com/concepts/environments)
```

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]

File: docs/src/_guide/troubleshooting.md [hunk 4]
```diff
@@ -122,3 +122,18 @@ Early versions of Meltano promoted a simple UI feature that was used for setting
...
+
+### Replacing UI functionality
+
+If you're using Airflow as your orchestrator, the Airflow webserver UI should be able to serve as a replacement for many Meltano UI use cases. See the ["Airflow orchestrator" section of the "Deployment in Production" guide](/guide/production#airflow-orchestrator) for more details on how to get the webserver running. From the webserver, you can [view all DAGs](https://airflow.apache.org/docs/apache-airflow/1.10.14/ui.html#dags-view). You can also access the Meltano logs for a specific task instance by going to the [task instance context menu](https://airflow.apache.org/docs/apache-airflow/1.10.14/ui.html#task-instance-context-menu) and clicking "Log" or "View logs".
```
Comment (aaronsteers) [inline docs/src/_guide/troubleshooting.md:139]:
```suggestion
If you're using [Airflow](https://hub.meltano.com/utilities/airflow) or [Dagster](https://hub.meltano.com/utilities/dagster) as your orchestrator, the Airflow or Dagster webserver UI should be able to serve as a replacement for many Meltano UI use cases. 

If using Airflow as orchestrator, see the ["Airflow orchestrator" section of the "Deployment in Production" guide](/guide/production#airflow-orchestrator) for more details on how to get the webserver running. From the webserver, you can [view all DAGs](https://airflow.apache.org/docs/apache-airflow/1.10.14/ui.html#dags-view). You can also access the Meltano logs for a specific task instance by going to the [task instance context menu](https://airflow.apache.org/docs/apache-airflow/1.10.14/ui.html#task-instance-context-menu) and clicking "Log" or "View logs".
```

File: docs/src/_guide/troubleshooting.md [hunk 5]
```diff
@@ -122,3 +122,20 @@ Early versions of Meltano promoted a simple UI feature that was used for setting
...
+The UI **does** work with:
+- Schedules based on [the `elt` command](https://docs.meltano.com/reference/command-line-interface#elt) (`meltano schedule add <schedule_name> --extractor <tap> --loader <target> --transform ...`)
+
+The UI does **not** work with:
+- Schedules based on [jobs](https://docs.meltano.com/reference/command-line-interface#job) (`meltano schedule add <schedule_name> --job <job>`)
+- [Environments](https://docs.meltano.com/concepts/environments)
+
+### Replacing UI functionality
+
+If you're using [Airflow](https://hub.meltano.com/utilities/airflow) or [Dagster](https://hub.meltano.com/utilities/dagster) as your orchestrator, the Airflow or Dagster webserver UI should be able to serve as a replacement for many Meltano UI use cases.
```
Comment (edgarrmondragon) [inline docs/src/_guide/troubleshooting.md:139]:
```suggestion
If you're using [Airflow](https://hub.meltano.com/utilities/airflow) or [Dagster](https://hub.meltano.com/utilities/dagster) as your orchestrator, the Airflow or Dagster webserver UI should be able to serve as a replacement for many Meltano UI use cases.
```

**Review by edgarrmondragon — APPROVED** [review]

**Review by afolson — APPROVED** [review]

**Comment by afolson** [comment]
@qbatten This looks great, thanks!

**Pull request merged**